lib/robots/dor_repo/speech_to_text/fetch_files.rb

```diff
@@ -23,11 +23,7 @@ def aws_provider
         end
 
         def sttable_filenames
-          @sttable_filenames ||= speech_to_text.filenames_to_stt
-        end
-
-        def speech_to_text
-          @speech_to_text ||= Dor::TextExtraction::SpeechToText.new(cocina_object:, workflow_context: workflow.context)
+          Dor::TextExtraction::SpeechToText.new(cocina_object:, workflow_context: workflow.context).filenames_to_stt
         end
```

Comment:
small refactor ... we don't really need this much indirection since the `filenames_to_stt` is only used once in the robot